python/conftest.py

```diff
@@ -0,0 +1,4 @@
+"""
+Test file containing shared test fixtures. Needed for now for test runner to pick up this folder.
+Will add fixtures as needed
```

Comment:
👀 are there supposed to be shared test fixtures in here? Is this file really necessary for pytest to find this directory? We already have files like `prairielearn_test.py` in this directory, and I believe they've been working fine.

Reply:
I tried removing it and had some issues, but I might have been doing something wrong. Will try again.

Reply:
So double checked, removing this file causes issues with the test runner being able to pick up imports from `prairielearn.py` when testing `big_o_utils.py`. Although kinda weird, I don't think this is an issue because (1) all the problematic imports will be gone once the desired functions get promoted to `prairielearn.py` and (2) this file is actually used to hold test fixtures in some of my other PRs (see https://github.com/PrairieLearn/PrairieLearn/pull/6764 for example).

As for the `conftest.py` semantics themselves, this is standard practice in pytest: https://docs.pytest.org/en/6.2.x/fixture.html